src/components/FormValidator.js

```diff
@@ -48,6 +48,10 @@ class FormValidator {
     this._submitButton.disabled = false;
   }
 
+  loadingButton() {
```

Comment:
Please remove this method from this class, the loading functionality should be implemented in `PopupWithForm` class and in `PopupWithConfirmation` class

You can add parameters `buttonText` and `loadingButtonText` to the constructor of `PopupWithForm` and using them to change button text. 

```js
constructor(popupSelector, { handleSubmit, buttonText, loadingButtonText }) { 
  …
  this._buttonText = buttonText;
  this._loadingButtonText = loadingButtonText;
}

showLoading() {
  this._submitBtn.textContent = this._loadingButtonText
}

hideLoading() {
  this._submitBtn.textContent = this._buttonText
}
```